Documentation/building/unix-test-instructions.md

```diff
@@ -4,48 +4,41 @@ Building and running tests on Linux, OS X, and FreeBSD
 CoreCLR tests
 -------------
 
-**Building**
+## Building
 
 Build CoreCLR on [Unix](https://github.com/dotnet/coreclr/blob/master/Documentation/building/linux-instructions.md).
 
-**Building the Tests**
+## Building the Tests
 
 DotNet is required to build the tests, this can be done on any platform then copied over if the arch or os does not support DotNet. If DotNet is not supported, [CoreFX](https://github.com/dotnet/corefx/blob/master/Documentation/building/unix-instructions.md) is also required to be built.
 
 To build the tests on Unix:
 
 > `./build-test.sh`
 
-To build on Windows:
-
-> `C:\coreclr>build-test.cmd`
-
 Please note that this builds the Priority 0 tests. To build priority 1:
 
 > `build-test.sh -priority 1`
 
+## Building Individual Tests
 
-**Building Individual Tests**
-
-During development there are many instances where building an individual test is fast and necessary. All of the necessary tools to build are under coreclr/Tools. It is possible to use coreclr/Tools/MSBuild.dll as you would normally use MSBuild with a few caveats.
+During development there are many instances where building an individual test is fast and necessary. All of the necessary tools to build are under `coreclr/Tools`. It is possible to use `coreclr/Tools/MSBuild.dll` as you would normally use MSBuild with a few caveats.
 
-Note that coreclr/Tools/msbuild.sh exists as well to make the call shorter.
+Note that `coreclr/Tools/msbuild.sh` exists as well to make the call shorter.
 
 **!! Note !! -- Passing /p:__BuildOs=[OSX|Linux] is required.** 
 
->If you omit it you will get the following error: `error MSB4801: The task factory "CodeTaskFactory" could not be loaded because this version of MSBuild does not support it.`
----
+Omitting the above will resulting in the following error: `error MSB4801: The task factory "CodeTaskFactory" could not be loaded because this version of MSBuild does not support it.`
```

Comment:
This is not true as of #19044 (I removed CodeTaskFactory)

Reply:
Although users still need to set BuildOS so it ends up in the correct location

Reply:
There is literally not enough +1's to describe how good that is to hear. Thanks!